### ISSUE TITLE
Template blocks don't work with Distant Horizons

### ISSUE THREAD

**Issue by Amzd** [quat1024/templates-mod#10]
Not sure if it should be reported here or on Distant Horizons but when blocks go out of render distance and it shows the lower detail DH blocks, they appear as missing texture:

![image](https://github.com/quat1024/templates-mod/assets/17001151/f3b139aa-0fb3-4190-985b-4dfbc5cfff96)
*Pink blocks are templates*

**Comment by quat1024** [comment]
I am not super familiar with DH. Had a look through [their public API](https://modrinth.com/mod/distanthorizonsapi) but couldn't figure out if there was a way to tell DH what color my blocks should be using information from a `BlockEntity`.

**Comment by Amzd** [comment]
I don't think this is as big of an issue anymore as DH ignores missing textures since 2.0.3 

https://gitlab.com/distant-horizons-team/distant-horizons/-/issues/385#note_1862081265

So at least no pink blocks anymore.
